platforms/android/app/src/main/assets/www/js/index.js

```diff
@@ -45,9 +51,64 @@ receivedEvent: function(id) {
 
 openSonect: function() {
     let credentials = {
-        token: "NDhkOGNiNTAtYjliNC0xMWU5LWJlNGMtNWRiNTMyOGNhZmE0OmY4ZWFhYzNlMGQyNjE5MjhiZGI5ZjQxNTk3NGVkMTU4MjA0YjQ5NTVhNzdjMDZiZTA1YTBjZWI3Nzk2NzExYzg=",
-        userId: "user1",
-        signature: "sUFLKPul8oTJZjBH/XvVhEv43zHs/F8fUbKuVEB0SFo="
+        token: "NGE1ODA5NjAtMDdiNS0xMWVhLWIyNzQtYjllZDdhMjk4ZTI5OjAzNjE4YWVjYTM1YzZmZDZlM2Q4YWExYTFjZTYwZWZlNTczZTY2NTNjNzMyMWMzYzVhMDliNDEzODQ3OThhZjg=",
```

Comment:
I've used new credentials from new partner. Could adjust to work in both android+ios later when whole API will be stable and cleared.